docs/upgrade/10_to_11/delete-old-indices.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+SERVER="http://localhost:9200"
+USER=""
+PASSWORD=""
+
+# indices
+ADMIN_UI_INDEX="adminui"
+EXTERNAL_API_INDEX="externalapi"
+
+# sub indices
+VERSION="version"
+EVENT="event"
+SERIES="series"
+THEME="theme"
+GROUP="group"
+
+set -e
+
+delete_indices() {
+	local INDEX="$1"
+	shift  # shift arguments to the left so we can get the rest as an array
+	local SUB_INDICES=("$@")
+	for SUB_INDEX in ${SUB_INDICES[@]}; do
+		DELETE_INDEX="${INDEX}_${SUB_INDEX}"
+		echo -e "Starting to delete ${DELETE_INDEX}.\n"
+
+		# Make sure index is open
+		echo -e "Making sure index is open:\n"
+		curl -u $USER:$PASSWORD -X POST  "$SERVER/$DELETE_INDEX/_open" -w "\n\n" -i -s
+
+		# Delete index
+		echo -e "Deleting index:\n"
+		RESPONSE=$(curl -u $USER:$PASSWORD -X DELETE  "$SERVER/$DELETE_INDEX" -i -s)
+		echo -e "${RESPONSE}\n"
+
+	        # Check for errors
+		if [[ $RESPONSE != *"200"* ]]; then
```

Comment:
Maybe use `curl -f …`?

Reply:
`-f` swallows the response body, and I thought that information might be helpful to some people. There is `--fail-with-body` now, but it's very new, my cURL for example can't do that yet...